data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3839 @@
+			`	"This is the monkey you speak so highly of? The one who fears not to come into our space, to fight our ships?" He asks the others, apparently mocking you.`
+			choice
+				`	"I fear nothing. Do you?"`
+					goto provoke
+				`	(Say nothing.)`
+				`	(Run away.)`
+					decline
+			`	"See? The monkey dares not even speak! He would not stand a chance in combat! Even our modest ship would bring him straight out of the sky! But we should not even try, or we could hurt the little monkey! But my friends, there is no shame for him to have... with a ship looking like that one I too would be hesitant to fight!" He seems very full of himself. Perhaps someone should teach him a lesson.`
+			choice
+				`	"You dare insult my <ship>?"`
+				`	"I have no need to prove myself to a rodent."`
+					goto provoke
+				`	(Go. He's not worth it.)`
+					decline
+			`	"Oh so that was your ship indeed! I was not sure, given your reputation, that you would fly in something like that. Your prowess is not exaggerated indeed then, for you managed to not crash on landing with such a thing," he says, and then he turns to address the crowd once again. "I wonder if he could take off again. Maybe he needs our assistance for that."`
+				goto lesson
+			label provoke
+			`	"So the monkey can speak? Let us see if it can fight as well," he says with a provocative tone.`
+			label lesson
+			`	Time for him to learn a lesson. But then it occurs to you that perhaps you are the one who needs to learn humility.`
+			`	After a cheering from his crowd, the arrogant Hai then speaks to you directly. "You and me, in battle, alone, a battle of flagships. Meet me in the skies, above the tallest trees, when you are ready. Except, of course, if you lack the courage to fight me. But rest assured, this is not to be a fight to the death, monkey."`
+			choice
+				`	"Let me show you my worth in battle. Right now."`
+					goto accept
+				`	(Crush his rodent face.)`
+					goto crush
+				`	"Can you tell me more about the rules of this fight?"`
+					goto rules
+				`	(Run away.)`
+			`	As you move away you hear laughter, but nobody tries to stop you. You have just shown to the Unfettered how much of a coward you are, and will probably never be offered any missions from them anymore.`
+				decline
+			label rules
+			`	"I guess you ought to know... You are supposed to bring only your flagship, but if it is too weak then more ships can be accepted, so long as it is a fair battle. It will last until either one of our ships is disabled, or one of us gives up. You can put it off for later at any given time for a symbolic bribe of 1000. Also, it is sometimes tolerated that the victor plunders the ashamed, but absolutely not that he tries to capture his ship. So, are you ready to fight?"`
```

Comment:
Its the start of the mission, it'd be hard to do that here, the bribe is really an actual bribe in space

Reply:
Cool. How about this?

```suggestion
			`	"I guess you ought to know... You are supposed to bring only your flagship, but if it is too weak then more ships can be accepted, so long as it is a fair battle. It will last until either one of our ships is disabled, or one of us gives up. You can put it off for later at any given time for a symbolic bribe of 1000 credits, offered in space before the battle begins. Also, it is sometimes tolerated that the victor plunders the ashamed, but absolutely not that he tries to capture his ship. So, are you ready to fight?"`
```

Reply:
I like it, but I'm thinking more along the lines of before the battle "truly" begins or smth like that, considering he'll be rushing to come and shoot you instantly x)
Also its not just before the battle begins it can be in battle, technically, if it gets too though